packages/test/local-server-tests/src/test/noDeltaStream.spec.ts

```diff
@@ -0,0 +1,130 @@
+import { LocalDeltaConnectionServer, ILocalDeltaConnectionServer } from "@fluidframework/server-local-server";
+import {
+    createAndAttachContainer,
+    createLoader,
+    ITestFluidObject,
+    OpProcessingController,
+    TestContainerRuntimeFactory,
+    TestFluidObjectFactory,
+} from "@fluidframework/test-utils";
+import { Container, DeltaManager } from "@fluidframework/container-loader";
+
+describe("No Delta Stream", () => {
+    const documentId = "localServerTest";
+    const documentLoadUrl = `fluid-test://localhost/${documentId}`;
+    const stringId = "stringKey";
+    const codeDetails: IFluidCodeDetails = {
+        package: "localServerTestPackage",
+        config: {},
+    };
+    const factory = new TestContainerRuntimeFactory(
+        "",
+        new TestFluidObjectFactory([[stringId, SharedString.getFactory()]]),
+        {generateSummaries: true});
+
+    let deltaConnectionServer: ILocalDeltaConnectionServer;
+    let opc: OpProcessingController;
+
+    async function createContainer(): Promise<IContainer> {
+        const loader: ILoader = createLoader(
+            [[codeDetails, factory]],
+            new LocalDocumentServiceFactory(deltaConnectionServer),
+            new LocalResolver());
+        const container = await createAndAttachContainer(
+            codeDetails,
+            loader,
+            createLocalResolverCreateNewRequest(documentId));
+        opc.addDeltaManagers(container.deltaManager);
+        return container;
+    }
+
+    async function loadContainer(noDeltaStream: boolean): Promise<IContainer> {
+        const loader: ILoader = createLoader(
+            [[codeDetails, factory]],
+            new LocalDocumentServiceFactory(deltaConnectionServer, noDeltaStream),
+            new LocalResolver());
+        const container = await loader.resolve({ url: documentLoadUrl });
+        opc.addDeltaManagers(container.deltaManager);
+        return container;
+    }
+
+    beforeEach(async () => {
+        deltaConnectionServer = LocalDeltaConnectionServer.create();
+        opc = new OpProcessingController();
+
+        // Create a Container for the first client.
+        const container = await createContainer();
+        const dataObject = await requestFluidObject<ITestFluidObject>(container, "default");
+
+        assert.strictEqual(container.deltaManager.active, false, "active");
+        assert.strictEqual(container.deltaManager.readonly, false, "readonly");
+
+        assert.strictEqual(dataObject.runtime.existing, false, "existing");
+        assert.strictEqual(dataObject.runtime.connected, true, "connected");
+        assert.notStrictEqual(dataObject.runtime.clientId, undefined, "clientId");
+
+        dataObject.root.set("test","key");
+        await opc.process();
+    });
+
+    it("Validate Properties on Loaded Container With No Delta Stream", async () => {
+        // Load the Container that was created by the first client.
```

Comment:
can we also add a case where we load storageOnly, then load 2 normal, do ops on the normal contianers and make sure it works, then finally load another storageOnly and ensure it sees the correct state